sub-packages/bionemo-amplify/src/bionemo/amplify/hf_rotary.py

```diff
@@ -0,0 +1,111 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: LicenseRef-Apache2
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Code copied from https://huggingface.co/chandar-lab/AMPLIFY_350M/blob/main/rotary.py
+# Original license:
+# MIT License
+
+# Copyright (c) [year] [fullname]
```

Comment:
This was the original license? Seems unlikely

Reply:
I would expect the person who wrote the code's name and whatever date they used, right?